sdk/attestation/azure-security-attestation/samples/attestation/attest_openenclave_with_draft_policy.cpp

```diff
@@ -21,14 +21,13 @@
  *
  */
 
-#include <get_env.hpp>
-
 #include "attestation_collateral.hpp"
 
 #include <azure/attestation.hpp>
 #include <azure/core/base64.hpp>
 
 #include <chrono>
+#include <get_env.hpp>
```

Comment:
I'd rather have it included with  `""` instead of `<>`, and in a separate incude group, but no strong opinion about this one.

Reply:
It comes from a separate package, and the rule is "include files from other packages use `<>`, include files from this package use `""`". 

For samples, the rule is modified to use `<>` for all files that aren't a part of the sample, because they are intended to mimic what customers would do in their own code.

We can change the rule if you'd like, but it's the rule :).